### PR TITLE
[full-ci][tests-only] bump middleware version to v1.1.0

### PR DESCRIPTION
**Pull request by kiranparajuli589**
### Description
- bump middleware version to v1.1.0 which contains `filesForUpload` dir
- so remove `filesForUpload` dir from the web repo
- make use of `filesForUpload` from middleware in web acceptance tests
- use middleware in e2e tests too

### Related Issue
- https://github.com/owncloud/web/issues/6209

### CODE REVIEW

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/21571
function waitForMiddlewareService accepts no arguments (1 given)

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait3</strong> https://drone.owncloud.com/owncloud/web/21695/45/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

File: .drone.star
Comment (phil-davis) [inline .drone.star:1351]: can be deleted if/when everything passes

File: .drone.star
Comment (phil-davis) [inline .drone.star:2384]: I guess this is helpful for easily debugging if something goes wrong with filesForUpload

File: .drone.star
Comment (phil-davis) [inline .drone.star:2900]:
```suggestion
        "image": "owncloud/owncloud-test-middleware:1.1.0",
```

Before merging, this should specify the tag that will soon be added to the middleware repo.

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md
Comment (SwikritiT) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:376]:
```suggestion
-   [webUITrashbinDelete/trashbinDelete.feature:85](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L85)
-   [webUITrashbinDelete/trashbinDelete.feature:71](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L71)
-   [webUITrashbinDelete/trashbinDelete.feature:48](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUITrashbinDelete/trashbinDelete.feature#L48)

```

File: .drone.star
Comment (kiranparajuli589) [reply .drone.star]: fixed

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md
Comment (kiranparajuli589) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md]: fixed

**Review by SwikritiT — COMMENTED** [review]
LGTM if CI passes

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/web/21765/15/20
https://github.com/owncloud/web/issues/6097
`webUISharingAcceptSharesToRoot/acceptShares.feature:108` failed.
I will restart CI.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**
